pkg/database/vercode.go

```diff
@@ -327,56 +330,63 @@ func (db *Database) UpdateStats(ctx context.Context, codes ...*VerificationCode)
 
 	// If the issuer was a user, update the user stats for the day.
 	if v.IssuingUserID != 0 {
-		sql := fmt.Sprintf(`
+		sql := `
 			INSERT INTO user_stats (date, realm_id, user_id, codes_issued)
-				VALUES ($1, $2, $3, 1)
+				VALUES ($1, $2, $3, $4)
```

Comment:
Ah this was a bug wasn't it.

Reply:
yes - this is a possible source of undercounting